testsupport/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <groupId>net.sourceforge.owlapi</groupId>
+    <version>3.5.2-SNAPSHOT</version>
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
My personal preference is to put modelVersion at the top so that it isn't confused for version.